style.json

```diff
@@ -120,12 +120,27 @@
       "filter": ["==", "class", "school"],
       "paint": {"fill-color": "rgb(236,238,204)"}
     },
+    {
+      "id": "waterway_tunnel",
+      "type": "line",
+      "source": "openmaptiles",
+      "source-layer": "waterway",
+      "filter": ["all", ["==", "brunnel", "tunnel"]],
+      "layout": {"visibility": "visible"},
```

Comment:
```suggestion

```

Could you please remove this line, it's not needed and we recently cleaned up the style in https://github.com/maputnik/osm-liberty/pull/61 :smile:

Reply:
done ;)